llama_stack/providers/inline/meta_reference/eval/eval.py

```diff
@@ -47,6 +52,19 @@ async def initialize(self) -> None: ...
 
     async def shutdown(self) -> None: ...
 
+    async def list_eval_tasks(self) -> List[EvalTaskDef]:
+        # NOTE: In order to be routed to this provider, the eval task def must have
+        # a EvalTaskDef with identifier defined as DEFAULT_EVAL_TASK_IDENTIFIER
+        # for app eval where eval task benchmark_id is not pre-registered
+        eval_tasks = [
+            EvalTaskDef(
+                identifier=DEFAULT_EVAL_TASK_IDENTIFIER,
```

Comment:
app eval don't have benchmark_id or any pre-registered task_id, we need to specify a default identifier in order for the router to route to this provider.

Reply:
what other EvalTaskDefs will this provider have? will you be able to dynamically register them? yes - right?

Reply:
Yes, they can be dynamically registered (registering benchmark tasks in https://github.com/meta-llama/llama-stack/pull/392)